Release ble controller 0.3.0-3.prealpha (#90)
NordicPlayground/nrfxlib: ble_controller/CHANGELOG.rst

All notable changes to this project are documented in this file.

ble_controller 0.3.0-3.prealpha
*******************************

Added
=======

> * nRF52833 is now supported.

Reviewer: ```suggestion
* Added support for nRF52833.
```

Author: Fixed

> * nRF52833 is now supported.

Reviewer 2: I would remove this entry. You have a different commit that fixes the issue with the build files. And since the support wasn't added in this release it seems wierd to mention it here. And the "Added" segment for the 0.3.0-2 release should cover it.

Author: Discussed offline. Should have been "Added" instead of "Changes". Now it is more clear

Bugfixes
========

> * Fixed an issue where ``hci_data_get()`` could return "No data available" when there was data available.

Reviewer: ```suggestion
* Fixed an issue where :cpp:func:`hci_data_get()` could return "No data available" when there was data available.
```

Author: Fixed, also in other occurences

> * Fixed an issue where ``hci_data_get()`` could return "No data available" when there was data available.

Reviewer 2: Should be `hci_data_packet_get`.

Author: No, it is `hci_data_get()`, see https://github.com/NordicPlayground/nrfxlib/blob/master/ble_controller/include/ble_controller_hci.h#L123

Reviewer 2: Ok. was looking at the wrong header. Thank you.

>   This could only occur when connected to multiple devices at the same time.

Reviewer: ```suggestion
  This issue would only occur when connected to multiple devices at the same time.
```

Author: Fixed
